dm/syncer/dml.go

```diff
@@ -419,9 +420,24 @@ func genDMLsWithSameTable(op sqlmodel.DMLType, jobs []*job) ([]string, [][]inter
 				args = append(args, arg)
 				continue
 			}
-			query, arg := j.dml.GenSQL(op)
-			queries = append(queries, query)
-			args = append(args, arg)
+
+			if i == 0 {
+				lastTable = j.dml.TargetTableID()
+			}
+			if lastTable != j.dml.TargetTableID() {
+				query, arg := genDMLsWithSameCols(op, groupDMLs)
+				queries = append(queries, query...)
+				args = append(args, arg...)
+
+				groupDMLs = groupDMLs[0:0]
+				lastTable = j.dml.TargetTableID()
+			}
+			groupDMLs = append(groupDMLs, j.dml)
```

Comment:
Can we combine these lines with L447?

Reply:
Will refine the code in future. In this sprint plan to let CDC use this library with @asddongmen